src/libraries/System.Console/tests/ReadAndWrite.cs

```diff
@@ -280,6 +281,14 @@ public static unsafe void OutputEncodingPreamble()
         }
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.Browser)]
+    public static unsafe void OutputEncodingPreamble_Browser()
+    {
+        Encoding curEncoding = Console.OutputEncoding;
```

Comment:
The test is primarily about testing Console.OutputEncoding.GetPreamble(), which this isn't doing.

Reply:
yeah, it's just OutputEncoding_Browser